scripts/appimage/build_appimage.sh

```diff
@@ -74,4 +74,8 @@ linuxdeploy-x86_64.AppImage --appdir appdir --plugin qt \
     -d "./appdir/usr/share/applications/com.kdab.hotspot.desktop" \
     --output appimage
 
+# package appdir with type 2 runtime so we don't depend on glibc and fuse2
+appimagetool-x86_64.AppImage --runtime-file /opt/runtime-x86_64 appdir
+
+mv Hotspot-x86_64.AppImage "/output/hotspot-static-$gitversion-x86_64.AppImage"
 mv Hotspot*x86_64.AppImage "/output/hotspot-$gitversion-x86_64.AppImage"
```

Comment:
why do we need to retain this image? isn't it sufficient to only keep the static version?

Reply:
it was just an idea in case the new one creates problems